### PR TITLE
Spp 9679 create technical spec for thousand pounds correction

### PR DESCRIPTION
**Pull request by jhmjam**
This PR is to create the technical specification for the thousand pounds method.

The totals and components specification also requires requires some formula changes which is handled here.

### CODE REVIEW

**Review by JasonBellONS — DISMISSED** [review]

File: editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+* unique_identifier: (String) - Unique identifier
+* principal_variable: (Float) - Numeric value that the method is working on
+* predictive: (Float) - Optional - Numeric value used for comparison.
+A previous 'valid' value (i.e. Returned/Imputed/Constructed)
+* auxiliary: (Float) - Optional - Alternative numeric` value used when
+a predictive value is not available and required by the user
+* upper_limit: (Float) - Upper bound of 'error value' threshold
+* lower_limit: (Float) - Lower bound of 'error value' threshold
+* target_variables: (List of Variables) - Optional - List of linked
```
Comment (JasonBellONS) [inline editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md:107]: List of float may be more accurate

File: editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+and
+
+```bash
+    auxiliary = None
+```
+
+If the predictive and auxiliary are None then we return a tailored
+error message. However, if both are 0 or predictive is 0 and
+auxiliary is none then and stop the method with an "S" TPC marker.
```
Comment (JasonBellONS) [inline editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md:141]: also an S is thrown if predictive is none and aux is 0

File: editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md (hunk 1 above)
Comment (jhmjam) [reply editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md:107]: Done

File: editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md (hunk 2 above)
Comment (jhmjam) [reply editing_and_imputation/editing/thousand_pounds_correction/technical_specification.md:141]: Done

**Review by JasonBellONS — DISMISSED** [review]
looks good to me

**Review by JasonBellONS — APPROVED** [review]
looks good to me

**Pull request merged**